packages/models/src/eservice/eservice.ts

```diff
@@ -71,6 +73,27 @@ export type DescriptorRejectionReason = z.infer<
   typeof DescriptorRejectionReason
 >;
 
+export const TemplateInstanceInterfaceMetadata = z.object({
+  name: z.string(),
+  email: z.string(),
+  url: z.string(),
```

Comment:
are these fields used in the `contact` section?
If yes, I suggest
```suggestion
  contactName: z.string(),
  contactEmail: z.string(),
  contactUrl: z.string(),
```

Reply:
handled in this https://github.com/pagopa/interop-be-monorepo/pull/1620